lib/src/scene.dart

```diff
@@ -132,4 +132,9 @@ abstract class Scene {
   /// **and call [rebuildScene()] afterwards.**
   List<EnvironmentControlBuilder> environmentControlBuilders =
       <EnvironmentControlBuilder>[];
+
+  /// Called when [EnvironmentControlPanel]'s "reset" button is pressed.
```

Comment:
Maybe call out that this is called within setState just so people understand what is going on, e.g. that the controls will be rebuilt based on the new state?

Reply:
Done.